v2/client.go

```diff
@@ -212,3 +214,58 @@ func (c *Client) UserNameLookup(ctx context.Context, usernames []string, opts Us
 		Raw: raw,
 	}, nil
 }
+
+func (c *Client) TweetRecentSearch(ctx context.Context, query string, opts TweetRecentSearchOpts) (*TweetRecentSearchResponse, error) {
```

Comment:
Any commentary you want to add to this exported func?

Reply:
yeah, working on this.  I make sure to run `golint` before merge to master